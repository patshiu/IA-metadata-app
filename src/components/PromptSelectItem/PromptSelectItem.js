import React from 'react';

const PromptSelectItem = () => {
  return (
    <div className="row center">
      <p>Please select an item from the dropdown.</p>
    </div>
  );
};

export default PromptSelectItem;
