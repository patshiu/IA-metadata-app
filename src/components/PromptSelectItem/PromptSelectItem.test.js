import React from 'react';
import renderer from 'react-test-renderer';

import PromptSelectItem from './PromptSelectItem';

it('renders prompt to select an item correctly', () => {
  const tree = renderer.create(<PromptSelectItem />).toJSON();
  expect(tree).toMatchSnapshot();
});
