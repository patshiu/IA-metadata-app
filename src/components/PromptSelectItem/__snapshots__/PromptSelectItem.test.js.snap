// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders prompt to select an item correctly 1`] = `
<div
  className="row center"
>
  <p>
    Please select an item from the dropdown.
  </p>
</div>
`;
